common/utils.go

```diff
@@ -323,3 +324,23 @@ func GetAzureCloud(url string) (string, error) {
 
 	return strings.TrimSpace(string(bodyBytes)), nil
 }
+
+func GetExecutableDirectory() (string, error) {
+	var (
+		dir string
+		ex  string
+		err error
+	)
+	ex, err = os.Executable()
+	if err == nil {
+		dir = filepath.Dir(ex)
```

Comment:
can you add a comment on why are we doing filepath.dir on nil case and EvalSymlinks on err case

Reply:
done